.github/workflows/continous-integration.yml

```diff
@@ -70,10 +71,9 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           checkName: Test Documentation
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
-          owner: ${{ github.event.pull_request.head.repo.owner.login || github.actor }}
           timeoutSeconds: ${{ env.WAIT_TIMEOUT_SECS }}
```

Comment:
in the run you reference (https://github.com/RasaHQ/rasa/runs/2279118028#step:2:166) it exits after timing out without failing, do we need to change that as well (great job catching it anyway btw!)

Reply:
> With regards to
> 
> > Remove the previous fix since we don't need to check docs in most of the PRs. Restrict the condition to run the wait for docs tests step as it is only relevant to docs changes. The step runs only on doc changes, or on tag pushes and pushes to the main branch exclusively, as it is a dependency of release jobs.
> 
> Does that mean it will fail if someone tries to update documentation from a fork? This is one of the most common community contributions

Yes, it will fail if someone updated the documentation and opened a pull request directly from their fork. But if the pull request got updated with the base branch, it will pass. 🤷🏻‍♀️

Reply:
> in the run you reference (https://github.com/RasaHQ/rasa/runs/2279118028#step:2:166) it exits after timing out without failing, do we need to change that as well (great job catching it anyway btw!)

We can catch the conclusion of this step and fail the step if the conclusion is not `success`.